app/managers.py

```diff
@@ -0,0 +1,28 @@
+import sqlite3
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self.connect = sqlite3.connect("../cinema.sqlite")
+
+    def create(self, first_name: str, last_name: str) -> None:
+        sql = "INSERT INTO actors" \
```

Comment:
Avoid using line breakers. It is a common practice to use triple quotes to write sql queries in python. Fix below as well